src/lib.rs

```diff
@@ -54,6 +51,7 @@ pub unsafe extern "C" fn getrandom(
     };
     buflen = buflen.min(maxlen);
 
+    let _ = ctru_sys::psInit();
```

Comment:
should we handle errors here and return `-1` with `errno` like we do if the random byte generation fails?